Database/CleanArchive.py

```diff
@@ -88,12 +96,69 @@ def clean(all=True):
         transaction = {
             '$pull': {'snippets': {'transcript': ''}}
         }
+        if duplicate < 1.0:
+            print('Duplicate found')
+            set_fields['duplicate'] = True
+            duplicates['metadata.Subtitle'] += 1
```

Comment:
The duplicate counter is going to have a LOT of fields. Maybe counting by series (metadata.Title) or network (metadata.Network) would give more readable results to the user.
I think instead of a duplicate field, we should add a duplicate error flag. See the other rules, but its something like errors.append('is_duplicate').
Then maybe add a field giving the list of shows it could be a duplicate of.